test/transaction-manager-test.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable max-len */
+/* eslint-disable comma-spacing */
```

Comment:
This package should handle comma-spacing automatically: https://www.npmjs.com/package/ripple-format